In [9]:
import numpy as np

1. Известно, что генеральная совокупность распределена нормально со средним квадратическим отклонением, равным 16. Найти доверительный интервал для оценки математического ожидания a с надежностью 0.95, если выборочная средняя M = 80, а объем выборки n = 256.

Так как известно среднее квадратичное отклонение генеральной совокупности, используем критерий Z.

X ± Zα/2 * σ/√n
где X = 80,
Zα/2 = 1,645
σ = 16
√n = √256

In [5]:
LX = 80 - 1.645*(16/pow(256, 0.5))
LX

78.355

In [7]:
UX = 80 + 1.645*(16/pow(256, 0.5))
UX

81.645

Доверительный интервал [78.355 - 81.645] с надежностью 0.95

2. В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью, получены опытные данные: 6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1 Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей, оценить истинное значение величины X при помощи доверительного интервала, покрывающего это значение с доверительной вероятностью 0,95. 

Так как неизвестно среднее квадратичное отклонение генеральной совокупности, используем критерий Стьюдента.
X ± tα/2 * σ/√n-1

In [10]:
##a = [6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1]
##σ = np.std(a)


0.42766809560686203

In [30]:
a = [6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1]
σ = np.std(a, ddof = 1)
σ


0.4508017549014448

In [ ]:
tα/2 = 2,262

In [36]:
X = np.mean(a)
X

6.590000000000001

In [40]:
LX = 6.59 - 2.262 * (0.4508/pow(10, 0.5))
LX

6.267539511206077

In [41]:
UX = 6.59 + 2.262 * (0.4508/pow(10, 0.5))
UX

6.912460488793923

In [ ]:
Доверительный интервал [6.2675 - 6.9125] с надежностью 0.95

3. Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм. Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметр оказался равным 17.5 мм, а дисперсия известна и равна 4 кв.мм.

M > M0, так критерий α односторонний

M0= 17
M =17.5
σ = √4 = 2
n = 100

In [42]:
Z1 = (17.5-17)/(2/10)
Z1

2.5

In [ ]:
Z0 =1,645
Z1 = 2.5
Z1 > Z0
Z лежит за пределом диапазона Z0 принятого критерия α гипетезы H0. Значит гипотеза H0 неверна.

4. Продавец утверждает, что средний вес пачки печенья составляет 200 г
Из партии извлечена выборка из 10 пачек. Вес каждой пачки составляет:
202, 203, 199, 197, 195, 201, 200, 204, 194, 190.
Известно, что их веса распределены нормально.
Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%?

In [46]:
a = [202, 203, 199, 197, 195, 201, 200, 204, 194, 190]
M = np.mean(a)
M

198.5

In [47]:
σ  = np.std(a, ddof=1)
σ

4.453463071962462

M < M0
-Z0(0.99) = - 2.33

In [49]:
Z1 = (198.5 - 200)/(4.45346/pow(10, 0.5))
Z1

-1.0651081384479864

In [ ]:
Z1 > Z0 
Гипотеза H0 верна.